### DISCUSSION TITLE
iterate کردن پیام های قبلی بعد از متوقف شدن ربات جهت پردازش پیام های جدید

### DISCUSSION THREAD

**Discussion by imanjahanyan** [kianahmadian/python-bale-bot#87] [Q&A] [answered]
سلام
ممکنه به هر دلیلی ربات چند دقیقه ای متوقف بشه و عملکرد نداشته باشه
بعد از راه اندازی مجدد همه پیام هایی که تو این فاصله ارسال شده رو پردازش میکنه؟
یا پایداری قابل اتکایی تو این موضوع نداره؟
اگه پایداری کافی نیست، راهی هست با استفاده از این پکیج بشه پیام های قبلی رو iterate و پردازش کرد؟
ممنونم بابت پاسخگویی شما

**Comment by kian-ahmadian** [comment, marked as answer]
سلام عزیز وقتتون بخیر
بله؛ در صورتی که در ربات یک مدت غیر فعال بشود و فرآیند `polling` رخ ندهد، در صورت فعال شدن دوباره ربات، شاهد دریافت آپدیت ها از آخرین آپدیت خوانده نشده تا به اکنون هستیم
